db/primary_migrate/20250228141538_add_timestamps_to_recaptcha_assessments.rb

```diff
@@ -0,0 +1,7 @@
+class AddTimestampsToRecaptchaAssessments < ActiveRecord::Migration[8.0]
+  def change
+    add_timestamps :recaptcha_assessments
+    change_column_comment :recaptcha_assessments, :created_at, from: nil, to: 'sensitive=false'
+    change_column_comment :recaptcha_assessments, :updated_at, from: nil, to: 'sensitive=false'
```

Comment:
Normally I would have expected to be able to add the comment as part of the `add_timestamps` call:

```suggestion
    add_timestamps :recaptcha_assessments, comment: 'sensitive=false'
```

But this was causing an error during migration, and it looks like the SQL query was including a stringified Ruby proc. Maybe a bug with `add_timestamps`? I had even tried it with a `change_table { |t| t.add_timestamps ... }` and got the same result.

```
ALTER TABLE "recaptcha_assessments" ADD "created_at" timestamp(6) NOT NULL, #<Proc:0x00000001476d1120 /gems/activerecord-8.0.1/lib/active_record/connection_adapters/postgresql/schema_statements.rb:1048>, ADD "updated_at" timestamp(6) NOT NULL, #<Proc:0x00000001476d0d38 /gems/activerecord-8.0.1/lib/active_record/connection_adapters/postgresql/schema_statements.rb:1048>
```